.github/workflows/deploy.yml

```diff
@@ -98,7 +142,7 @@ jobs:
         run: python -m pip install pytest && python -m pytest -m "not accessibility" .
 
   deploy_dev:
-    if: ${{ always() && (needs.testing-unit.result=='success' || needs.testing.result=='success')}}
+    if: ${{ always() && (needs.testing-unit.result=='success' || needs.testing.result=='success' || needs.testing-unit-postgres.result=='success') && inputs.deploy_to_dev == true}}
```

Comment:
Same here about the `always()` - also if either of those jobs has failed, won't it skip later jobs anyway?

Reply:
thats right, its the only way to have blocking jobs that dont make the job conditional